README.md

```diff
@@ -1,6 +1,12 @@
 # Pan Canadian Genome Library Data Access Compliance Office
 
-> <img src="./apps/ui/public/pcgl-logo.png" height="90" />
+<div>
```

Comment:
sheer curiosity: how does this div make any difference?

Reply:
It's implemented this way in Stage: https://github.com/overture-stack/stage/blob/develop/README.md?plain=1

TBH I don't believe it makes a layout difference? At least not in VSCode Markdown preview

Reply:
how curious! if it doesn't add, let's remove it then

Reply:
Removed